app/_src/gateway/plugin-development/wasm.md

```diff
@@ -0,0 +1,111 @@
+---
+title: Create a Proxy-Wasm filter
+book: plugin_dev
+chapter: 11
+---
+
+## Your first Proxy-Wasm filter
+
+Filters can be written as a compilation target in almost any language that supports WebAssembly, but the easiest way to get started is by
+using a language which has an existing [Proxy-Wasm](https://github.com/proxy-wasm/spec)
+SDK, such as [Go](https://github.com/tetratelabs/proxy-wasm-go-sdk/)
+or [Rust](https://github.com/proxy-wasm/proxy-wasm-rust-sdk/).
+
+## Coding, building and deploying a Proxy-Wasm filter
+
+Kong made Go and Rust Proxy-Wasm filter templates available, they can be used
+to start writing your own filters, or to build as they are and get your first `hello
+world`-type filter:
+
+* [Go Proxy-Wasm filter template](https://github.com/Kong/proxy-wasm-go-filter-template/)
+* [Rust Proxy-Wasm filter template](https://github.com/Kong/proxy-wasm-rust-filter-template/)
+
+### Deploying the filter with Kong
+
+After writing and building the filter, deploy it:
```

Comment:
It's a bit weird to "colon into a section", at first glance it felt like something was missing. A short summary might work better:

```suggestion
After writing and building the filter, the next steps are to configure Kong to enable Wasm support and add the filter, then associate the filter to a route or a service. We will then be able to start Kong and issue requests that go through the filter.
```